packages/app/src/runner/SpecRunner.vue

```diff
@@ -97,12 +106,27 @@ mutation OpenFileInIDE ($input: FileDetailsInput!) {
 }
 `
 
+gql`
+mutation SpecRunner_SetPreferences ($value: String!) {
+  setPreferences (value: $value)
+}`
+
+const props = defineProps<{
+  gql: SpecRunnerFragment
+  activeSpec: BaseSpec
+}>()
+
 const eventManager = getEventManager()
 
 const autStore = useAutStore()
 const screenshotStore = useScreenshotStore()
 const runnerUiStore = useRunnerUiStore()
 
+runnerUiStore.setAutoScrollingEnabled(props.gql.localSettings.preferences.autoScrollingEnabled ?? true)
```

Comment:
I foresee a bunch of default values coming in the future, we might want a `setDefaults` function at some point but prob fine for now. Good to keep in mind if this pattern of setting defaults expands.